redcap_bridge/utils.py

```diff
@@ -122,3 +122,46 @@ def remove_columns(csv_file, compressed_file=None):
 def exportCSVtoXLS(csv_file, compressed_file=None):
     read_file = pd.read_csv(csv_file, na_filter=False, dtype='str')
     read_file.to_excel(r'Path', index=None, header=True)
+
+
+def conversion_csv_to_json(csv_file):
+    """
+    Test conversion function
+    """
+    df = pd.read_csv(csv_file)
+    df.to_json(
+        r'/Users/killianrochet/PycharmProjects/DigLabTools/redcap_bridge/test_redcap/testfiles/elabConversion/pandaJsonConversion..json')
+    json_df = pd.read_json(
+        '/Users/killianrochet/PycharmProjects/DigLabTools/redcap_bridge/test_redcap/testfiles/elabConversion/pandaJsonConversion..json')
+    for i in json_df.columns:
+        for j in json_df.index:
+            if json_df[i][j] == 'dropdown':
+                json_df[i][j] = dropdown_to_json()
+            elif json_df[i][j] == 'text':
+                json_df[i][j] = text_to_json()
+            elif json_df[i][j] == 'notes':
+                json_df[i][j] = notes_to_json()
+            elif re.search(r'@DEFAULT=.', str(json_df[i][j])):
+                print(f"MATCH")
+                json_df[i][j] = default_value_to_json()
+            else:
+                print(f"Other type")
+
+
+
+def text_to_json():
```

Comment:
These functions should then take the dict representation of a single row, e.g. 
```suggestion
def text_to_json(field_dict):
"""
Convert a RedCap text field dictionary into a ElabFTW field dictionary.

    Args:
        field_dict: dictionary containing RedCap definition of a survey text field
    Returns:
        dict
            ElabFTW representation of the survey field
"""
```